frontend/src/utils/propertyUtils.ts

```diff
@@ -17,6 +17,26 @@ export const pidFormatter = (pid?: string) => {
   return '';
 };
 
+/**
+ * The pidParser is used to return a numeric pid value from a formatted pid.
+ * @param {string} pid This is the target PID to be parsed
+ */
+export const pidParser = (pid?: string): number | undefined => {
+  if (typeof pid === 'number') {
```

Comment:
is it possible that the pid is a number? if so, the method definition should be ```pidParser = (pid?: string | number)```

Reply:
agree typing could be improved here